jeoparpy/ui/maingame/podiapanel.py

```diff
@@ -76,24 +76,24 @@ def _init_background(self):
         img = pygame.transform.scale(img, self.size)
         self.blit(img, (0, 0))
 
-        return sizeScalar
+        return sizeScalar * .75
 
     def _init_podia(self, gameData, scalar):
```

Comment:
@Agent-E11 could you adjust this formula so that is referenced to the screen resolution instead of hardcoded? Factor in the number of players as it will be variable based on the player.txt file now.

Reply:
I will merge this first, and then do this on my own fork